sdk/nodejs/src/client.ts

```diff
@@ -59,13 +56,15 @@ export class DevCycleClient {
     private clientUUID: string
     private hostname: string
     private sdkKey: string
-    private configHelper: EnvironmentConfigManager
+    private configHelper?: EnvironmentConfigManager
```

Comment:
why does this need to be optional?

Reply:
it isnt initialized until the bucketing import resolves